app/form_objects/age_range_form.rb

```diff
@@ -0,0 +1,94 @@
+class AgeRangeForm
+  include ActiveModel::Model
+
+  attr_accessor :age_range_in_years, :presets
+  attr_reader :course_age_range_in_years_other_from,
+              :course_age_range_in_years_other_to
+
+  def course_age_range_in_years_other_from=(value)
+    @course_age_range_in_years_other_from = if value.present?
+                                              value.to_i
+                                            else
+                                              value
+                                            end
```

Comment:
Would it be better to have the `if` statement in a method called something like `get_value`. This could be used here and in the `course_age_range_in_years_other_to` method. Or perhaps shorthand? `(value.present?)? value.to_i : value`

Reply:
- I don't think `get_value` makes sense as that is passed into the method as it not available elsewhere
- Changing to a one-line ternary here I don't think really improves readability and would merely be for vanity sake

Reply:
having a hard time thinking what is `value` 

Maybe it should be 
```suggestion
    @course_age_range_in_years_other_from = value.to_i if value.present?
```

we got a lot of 

`course_age_range_in_years_other_from.blank?`
`course_age_range_in_years_other_from.present?`
`course_age_range_in_years_other_from.to_i`

Reply:
I was wanting to capture empty string which your version does not and instead would set to `nil`. That way you can run validations of what the user actually entered. The alternative is to make virtual attributes (another method and variable) for the user input and another set for a processed version that we can use later